src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--month-length--31">
+      <div class="calendar__cell calendar__cell--day--1"></div>
```

Comment:
```suggestion
      <div class="calendar__cell"></div>
```
Do you really need to hardcore a modifier for each day? You can use for loop to set content inside each day according to the index